server/models/user.js

```diff
@@ -3,10 +3,11 @@ const Schema = mongoose.Schema;
 
 // Placeholder User schema
 const UserSchema = new Schema({
-  firstname: { type: String, required: true },
-  lastname: { type: String, required: true },
+  first_name: { type: String, required: true },
+  last_name: { type: String, required: true },
   email: { type: String, required: true, unique: true },
   password: { type: String, required: true },
+  gmail_token: { type: String },
```

Comment:
because this is mongoDB and it speaks JSON, we should use camelCase for the field names (https://intellipaat.com/community/28608/what-are-naming-conventions-for-mongodb)

Reply:
Will change.

@airensurzyn worth noting as I was following the style from the Prospect model document so it's probably best we change everywhere :)

Reply:
@umbauk yes, I have changed it in my prospect model in my PR. Thanks for the heads up